shader/antialiasing/cmaa2/cmaa2_common.glsl

```diff
@@ -0,0 +1,176 @@
+// create several presets for defines in shader setup
+#define CMAA_PACK_SINGLE_SAMPLE_EDGE_TO_HALF_WIDTH  1
+#define CMAA2_PROCESS_CANDIDATES_NUM_THREADS        128
+#define CMAA2_DEFERRED_APPLY_NUM_THREADS            32
+#define CMAA2_DEFERRED_APPLY_THREADGROUP_SWAP       1 
+#define CMAA2_USE_HALF_FLOAT_PRECISION              0
+#define CMAA2_SUPPORT_HDR_COLOR_RANGE               0
+#define CMAA2_EDGE_DETECTION_LUMA_PATH              0       // We should use HDR luma from a separate buffer in the future
+#define CMAA2_EDGE_UNORM                            0
+#define CMAA2_EXTRA_SHARPNESS                       0
+
+const float c_symmetryCorrectionOffset = 0.22;
+
+#if CMAA2_EXTRA_SHARPNESS
+  const float c_dampeningEffect = 0.11;
+  const float g_CMAA2_LocalContrastAdaptationAmount = 0.15f
+  const float g_CMAA2_SimpleShapeBlurinessAmount = 0.07f
+#else
+  const float c_dampeningEffect = 0.15;
+  const float g_CMAA2_LocalContrastAdaptationAmount =  0.10f;
+  const float g_CMAA2_SimpleShapeBlurinessAmount = 0.10f;
+#endif
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// VARIOUS QUALITY SETTINGS
+//
+// Longest line search distance; must be even number; for high perf low quality start from ~32 - the bigger the number, 
+// the nicer the gradients but more costly. Max supported is 128!
+const uint c_maxLineLength = 86;
+// 
+
+layout(binding = 0) uniform sampler2D g_inoutColorReadonly;
```

Comment:
`g_inoutColorReadonly` -> `sceneTonemapped` ?

Reply:
I've renamed to more general `inputColorReadonly`, since input could be in HDR potentially